src/main/java/com/amazon/opendistroforelasticsearch/sql/plugin/RestSqlAction.java

```diff
@@ -141,7 +142,8 @@ private void executeSqlRequest(final RestRequest request, final QueryAction quer
                                    final Client client, final RestChannel channel) throws Exception {
         if (isExplainRequest(request)) {
             final String jsonExplanation = queryAction.explain().explain();
-            channel.sendResponse(new BytesRestResponse(OK, "application/json; charset=UTF-8", jsonExplanation));
+            final String jsonPretty = (new JsonPrettyFormatter()).format(jsonExplanation);
```

Comment:
I wonder with this change do we pretty format all the time or only when `pretty` param given in url?

Reply:
Actually all the time because the output would be a oneline string, which I think no need to remain. But I guess we could add to a couple of other format options with `?pretty` param in url, if this specific enhancement is needed.

Reply:
> Actually all the time because the output would be a oneline string, which I think no need to remain. But I guess we could add to a couple of other format options with `?pretty` param in url, if this specific enhancement is needed.

Yeah, I think it would be great if we can do this only when `pretty` given which is consistent with behavior of other ES API, for example `_cluster/health`.

Reply:
It seems ES SQL doesn't provide pretty option for translate API neither. So we're good for now.